src/coins.h

```diff
@@ -275,7 +275,7 @@ class CCoinsViewCache : public CCoinsViewBacked
      * Failure to call this method before destruction will cause the changes to be forgotten.
      * If false is returned, the state of this cache (and its backing view) will be undefined.
      */
-    bool Flush();
+    bool Flush(bool clear_cache = true);
```

Comment:
Isn't passing the argument explicitly better than with a default one? Also it's worthy to extend method comment a bit with effect of argument on cache.